package.json

```diff
@@ -750,6 +750,21 @@
           "type": "array",
           "default": null,
           "description": "An array of strings that enable experimental features in the PowerShell extension."
+        },
+        "powershell.pester.pester5CodeLens": {
+          "type": "boolean",
+          "default": false,
+          "description": "Enables code lense that is specific to Pester 5. This enables Run Tests on every Describe / Context and It and other features."
+        },
+        "powershell.pester.output": {
```

Comment:
What do you think about calling this either:

* `"powershell.pester.outputStyle"`
* `"powershell.pester.outputVerbosity"`

Reply:
OutputVerbosity sounds good. And it is how it’s called in Pester 5. Will rename.